docs/01-overview/README.md

```diff
@@ -2,31 +2,39 @@
 title: What is Kyma?
 ---
 
-## Purpose
+Kyma is an opinionated set of Kubernetes-based modular building blocks, including all necessary capabilities to develop and run enterprise-grade cloud-native applications.
+It is the open path to the SAP ecosystem supporting business scenarios end-to-end.
 
-Kyma is an application runtime that provides you a flexible and easy way to connect, extend, and customize your applications in the cloud-native world of Kubernetes.
+Kyma is an actively maintained open-source project supported by SAP. The Kyma project is also a foundation of Kubernetes Runtime which is a part of SAP Business Technology Platform (BTP). You can use Kyma modules in your own Kubernetes cluster, or try the managed version from SAP BTP with a ready-to-use Kubernetes cluster powered by Gardener. 
```

Comment:
> I don't really get this sentence:
> 
> > The Kyma project is also a foundation of Kubernetes Runtime which is a part of SAP Business Technology Platform (BTP).
> 
> Are we trying to say that the (OS) Kyma project is the foundation of the SAP BTP, Kyma runtime?
> 
> Because "Kubernetes Runtime", capitalized and without article, looks like it's an SAP product name, but I don't know any product with that name. I only know of SAP BTP, Kyma runtime.

Good point, thank you!

Reply:
Good point, thank you!